charts/observability-pipelines-worker/ci/dupe-ports-values.yaml

```diff
@@ -1,3 +1,8 @@
+args:
+  - run
+  - --skip-key-validation
+  - --pipelines-config
+  - "/etc/observability-pipelines-worker/pipelines.yaml"
```

Comment:
@spencergilbert let me know if this is correct/if there's a way to avoid this duplicated override.

Reply:
I _think_ this is right, without needing to change the shared CI workflow. We can also look to remove some of the cases, they may not all be needed with the more streamlined OPW experience.